src/cli.rs

```diff
@@ -412,14 +410,12 @@ pub fn run(args: Opt) -> Result<(), Box<dyn Error>> {
             value,
             pkg_opts,
         } => {
-            if name == "name".to_owned() {
+            if name == "name" {
                 return Err("To change the name please use the rename command!".into());
             }
             let predicate = make_pkg_predicate(pkg_opts)?;
             let type_value = {
-                if &value == "true" {
-                    Value::from(true)
-                } else if &value == "false" {
+                if &value == "true" || &value == "false" {
                     Value::from(true)
```

Comment:
@gnunicorn Should there be `false` here instead?

Reply:
yes.